tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf

```diff
@@ -5,10 +5,10 @@
 # Account is currently registered with jhasenau@greenpeace.org
 
 # `AccountID` is from your MaxMind account.
-AccountID 146520
```

Comment:
File not required

Reply:
I moved this file into tests/src/planet-4-151612/openresty/geoip/etc/nginx/GeoIP.conf where it is needed now.